keys.js

```diff
@@ -0,0 +1,16 @@
+
+var Keys = {
+	twitterKeys: {
+		consumer_key: 'QoyDrcDDCYe4OeFrY4b7NNM2j',
+		consumer_secret: 'WpFqfoWTQAYtIMN2nEjc3zURXszwzayz0t0sroxO0XQ2XpmilE',
+		access_token_key: '333519507-cMGyzOVqoQwBVN0QONgBv0OkOLFyTLfMXkt5fwWJ',
+		access_token_secret: 'ZKTZNCGOdhgyJ38R7D1zT8Gpy1eWcEHfVeyMU5l9MjCO5'
+	},
+	spotifyKeys: {
+		client_id: 'e326999a21944e729a09cca3366ec451',
+		client_secret: 'c594d19819634ce5a8c253e6954d6f74'
+	}
+  
+};
+
+module.exports = Keys;
```

Comment:
I would create the instances of your api's instead of making them in your liri file.